src/ship/components/Board/Board.tsx

```diff
@@ -1,13 +1,17 @@
 import type { CSSProperties } from 'react';
+import { useRef } from 'react';
 import { useCallback, useMemo } from 'react';
+import { useDrop } from 'react-dnd';
 import classNames from 'clsx';
+import { get } from 'radash';
```

Comment:
`get` is ambiguous, can you rename the dependency ?
```suggestion
import { get as radashGet } from 'radash';
```

Reply:
:heavy_check_mark: